apps/docs/src/components/Icons/IconPreview.tsx

```diff
@@ -0,0 +1,21 @@
+import { h } from 'preact';
+
+import { Icon } from '@selleo/core/src/Icon';
+
+type IconPreviewProps = {
+  name: string;
+} & h.JSX.HTMLAttributes<HTMLDivElement>;
+
+export const IconPreview = ({ name, ...props }: IconPreviewProps) => {
+  return (
+    <div
+      class="flex flex-col justify-center items-center gap-2 rounded border-[1px] border-neutral-600 min-h-[120px] overflow-hidden p-[5px] cursor-pointer"
```

Comment:
no need to use such arbitrary values everywhere so please stick to spacers & grid for margins/paddings/borders

Reply:
also there is a built in class for border 1px
https://tailwindcss.com/docs/border-width

Reply:
Okay, done